changelogs/unreleased/2209-markrity

```diff
@@ -0,0 +1 @@
+Replaced deprecated helm repo url at all it appearance at docs.
```

Comment:
maybe a new line here?

Reply:
I have taken a look at all other unreleased changelogs, they look similar to this, so I didn't get the point of new line?

Reply:
I think it's fine as-is, and to be honest we don't usually require changelogs for docs changes anyway, so don't worry about this one.